### ISSUE TITLE
List supported methods (documentation)

### ISSUE THREAD

**Issue by petermilan** [jubos/fake-s3#65]
Hello, could you please be more specific in the documentation and add a concrete list of supported methods?

```
FakeS3 doesn't support all of the S3 command set, but the basic ones like put, get, list, copy, and make bucket are supported. More coming soon.
```

It seems that it is not clear what is supported

List Objects seems to return empty structure for me, and when I am trying to use https://www.npmjs.org/package/s3-upload-stream library it seems that  multipart upload is not working properly. Could there be a list of all supported methods perhaps according to the AWS-SDK?

**Comment by LinusU** [comment]
I would be very glad if we could clarify that multipart uploads are not supported. Just ran into an issue where uploads >5 MB would fail with a strange error. Turns out that it's because the AWS SDK automatically uses the multipart api for files over that size.

Fixing this would hopefully stop people like me from bothering the AWS developers with stuff like aws/aws-sdk-js#525.

**Comment by juanotto** [comment]
I agree with the need to improve the docs, its a really good project, it deserves better documentation.

For the multipart upload, it was added a couple of weeks ago and it works great with s3cmd. If using java check my pull req, it solves a small format difference from the java library.

**Comment by CpuID** [comment]
+1
